tests/braille-specs/ro-g0.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright © 2023 by BAUM Engineering <liblouis@baum.ro>
```

Comment:
Since you mark the table with `#+direction:both`, it would make sense to run the tests also in both directions. You can do this with `flags: {testmode: bothDirections}`.

Reply:
Done